src/components/Searchbar/Search.tsx

```diff
@@ -67,7 +90,14 @@ export const Search = () => {
             ) : (
               searchRes.map((res) => {
                 return (
-                  <li key={res.id} className="search__list-item">
+                  <li
```

Comment:
for click feature here you can set title of search to correct value to show related anime